x-pack/plugins/ml/server/models/data_recognizer/modules/security_cloudtrail/ml/rare_method_for_a_city.json

```diff
@@ -29,6 +29,7 @@
     "time_field": "@timestamp"
   },
   "custom_settings": {
-    "created_by": "ml-module-siem-cloudtrail"
+    "created_by": "ml-module-security-cloudtrail",
+    "security_app_display_name": "Unusual City For an AWS Command"
```

Comment:
Nit - would use lowercase `for` here - which would be consistent with what you have for the username job.